google-cloud-spanner/lib/google/cloud/spanner/project.rb

```diff
@@ -435,6 +437,21 @@ def create_database instance_id, database_id, statements: []
         #     becomes available. The default is `true`.
         #   * `:threads` (Integer) The number of threads in the thread pool. The
         #     default is twice the number of available CPUs.
+        # @param [Hash] session_labels The labels to be applied to all sessions
```

Comment:
The only bone I have to pick is with the choice of `session_labels` instead of `labels`. Can you help me understand why `session_labels` is a better choice?

Reply:
Yes. The `labels` name is perfect for `Project#create_instance` because the method clearly identifies the type that will receive the labels. However, `Project#client` and `#batch_client` do not in any way identify that `Session` will receive the labels. Although it is mentioned in the docs, the `session_labels` naming is important to clarify this for users.

Reply:
Fwiw, `sessionLabels` is also used in [SpannerOptions.java](https://github.com/vkedia/google-cloud-java/blob/0bec2b43b749ca630160760ce23562d0a1f3b377/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java#L161-L176).

Reply:
We use `pool` even though it is not specifically prefixed with `session_`. I am just a tad uncomfortable with the inconsistency.

Reply:
Changed to `labels`.